pages/index.tsx

```diff
@@ -1,18 +1,12 @@
 import { Box, Flex, H1, H4, Panel } from '@bigcommerce/big-design';
-import { useEffect } from 'react';
+import React from 'react';
```

Comment:
are we using the `React` variable anywhere? If not, I believe Next will automatically import this for us

Reply:
Ah that's a habit, forgot that's apart of how [React transforms JSX](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html)